pkg/analytics/client.go

```diff
@@ -78,16 +79,20 @@ func (t *Client) Debug(event string) {
 
 func (t *Client) Warn(event string) {
 	t.Properties[datadogLogLevel] = Warn
+	t.Properties[datadogStatus] = Warn
```

Comment:
were we always sending the logs, just classifying them as Info? I thought we send the "klotho compiling failed" as info, but also with the error logs and thats what was also missing